[Frontend V2] Fix alignment of visibility option on submit page

Alabhya268 opened the pull request:
fixed #3349

It seems I forget to add css when making #3229.

![submit](https://user-images.githubusercontent.com/57143358/112286257-23a44400-8cb1-11eb-9a46-5794c7f577b0.png)

Alabhya268 commented:
@Ram81, Please review this PR.

Ram81 commented on frontend_v2/src/app/components/challenge/challengesubmit/challengesubmit.component.scss:
```diff
@@ -122,3 +122,7 @@ th {
 }
+
+.submission_visibility {
+  margin-left: 1em;
```
@Alabhya268 let's not add a new css class for every change. Please put this change in existing css class or use one of the existing css class for this field.

Ram81 reviewed (changes requested):
@Alabhya268 the spacing between submission visibility and the next field is inconsitent with submission meta attributes field spacing. Can we fix that too.

Alabhya268 commented:
@Ram81, I've fixed submit page css issue using pre written css class.
same classes are used as submission ```meta attributes```.
This also fixed space issues between ```submission visibility``` and ```submission meta```.
Please review again.

![sss](https://user-images.githubusercontent.com/57143358/112587235-3e96c580-8e23-11eb-81de-5bb5092ac1c9.png)

Ram81 commented:
@Alabhya268 can you also share a screenshot of a challenge with additional submission meta attributes.

Alabhya268 commented:
> @Alabhya268 can you also share a screenshot of a challenge with additional submission meta attributes.

@Ram81, There is a bug in frontendv2, it not showing more submission attributes.

Frontentv1 screenshot
![2](https://user-images.githubusercontent.com/57143358/112629408-4cb40880-8e5a-11eb-9ac9-02c955aa4e8e.png)

Frontendv2 screenshot
![1](https://user-images.githubusercontent.com/57143358/112629472-60f80580-8e5a-11eb-9b0a-8497d92318a3.png)


As you can see frontendv2 does not show additional submission meta attributes.
This has nothing to do with this PR as I have also tested it on the master branch.

Alabhya268 commented:
@Ram81

Ram81 commented on frontend_v2/src/app/components/challenge/challengesubmit/challengesubmit.component.html:
```diff
@@ -236,7 +236,8 @@ <h5 class="fw-light">Make Submission</h5>
-                <div>
+                <div class="row row-lr-margin">
+                  <div class="align-left col-sm-10 col-lr-pad">
                   <p class="fs-14 w-400">Select submission visibility:</p>
```
@Alabhya268 Can you please indent the content of the `<div>` tag.

Alabhya268 replied:
> @Alabhya268 minor change

@Ram81, I have fixed the indent, Please look at this PR again.

Ram81 reviewed (changes requested):
@Alabhya268 minor change

Alabhya268 commented:
@Ram81, I have fixed the indent, Please look at this PR again.

Ram81 reviewed (approved):
LGTM 👍

Alabhya268 commented:
@RishabhJain2018
